core/templates/dev/head/pages/exploration_editor/editor_navigation_directive.html

```diff
@@ -84,7 +90,7 @@
     </a>
     <div ng-show="getOpenThreadsCount()" class="oppia-exploration-open-threads-indicator oppia-exploration-open-threads-color">
       <span class="oppia-exploration-open-threads-count">
-        <[getOpenThreadsCount()]>
+        <[getOpenThreadsCount()]>&nbsp;open
```

Comment:
Hm. Why not just a space character instead of `&nbsp;`?

Reply:
Sorry, I don't think I understand. What is the bad thing that happens if you used a space character?

Reply:
On using "&nbsp;" the whole string is treated as a word. and upon resizing or any other constraint they won't separate (as it's one whole word). It's just to prevent any such possibility that could cause the words to saperate

Reply:
OK, thanks for explaining.